Makefile

```diff
@@ -87,8 +87,8 @@ else
 endif
 
 install: go.sum check-ledger
-	go install -mod=readonly $(BUILD_FLAGS) ./cmd/gaiad
-	go install -mod=readonly $(BUILD_FLAGS) ./cmd/gaiacli
+	go install $(BUILD_FLAGS) ./cmd/gaiad
```

Comment:
mmmm how comes? Did you have any issues with `-mod=readonly`? Is there anything I can help out with?

Reply:
Sure! I was having issues with `-mod=readonly`!

Reply:
Why is this the case @alessio?